sql/core/pom.xml

```diff
@@ -69,6 +69,11 @@
       <version>2.3.0</version>
     </dependency>
     <dependency>
+      <groupId>org.jodd</groupId>
+      <artifactId>jodd-core</artifactId>
+      <version>${jodd.version}</version>
+    </dependency>
```

Comment:
I'm pretty hesitant to add a dependency here as they are very high cost for a project as big as Spark.  Is there any way to do this without adding this?